### PR TITLE
Fix use Asyncftpclient:retrfile error.

### PR DESCRIPTION
**Pull request by Angluca**
Can download file now.

### CODE REVIEW

File: lib/pure/asyncftpclient.nim [hunk 1]
```diff
@@ -293,17 +299,20 @@ proc retrFile*(ftp: AsyncFtpClient, file, dest: string,
   ## The ``EvRetr`` event is passed to the specified ``handleEvent`` function
   ## when the download is finished. The event's ``filename`` field will be equal
   ## to ``file``.
-  var destFile = open(dest, mode = fmWrite)
   await ftp.pasv()
-  var reply = await ftp.send("RETR " & file.normalizePathSep)
-  assertReply reply, ["125", "150"]
-  if {'(', ')'} notin reply.string:
-    raise newException(ReplyError, "Reply has no file size.")
+  var reply = await ftp.send("size " & file.normalizePathSep)
```
Comment (dom96) [inline lib/pure/asyncftpclient.nim]: Why is this necessary? The ``SIZE`` command is part of [RFC 3659](https://tools.ietf.org/html/rfc3659), what if the server doesn't support it?

What should be done instead is not failing when the file size is not specified by ``RETR``'s reply, instead just download the file until EOF. As described by section 3.4 of https://tools.ietf.org/html/rfc959

Comment (dom96) [inline lib/pure/asyncftpclient.nim]: More info here http://stackoverflow.com/questions/22151827/how-to-determine-the-end-of-a-file-when-using-retr-on-ftp-server-pasv

**Comment by dom96** [comment]
Can't accept this, sorry.

**Comment by Angluca** [comment]
Oh well, I don't know ftp have more protocol, Maybe I should change ftp protocaol test, thank u.

**Pull request closed without merging**